src/main/java/doit/apihw/api/controller/MemberController.java

```diff
@@ -50,6 +46,10 @@ public void changePassword(@PathVariable Long memberId, @RequestBody AuthPasswor
         memberService.changePassword(memberId, request);
     }
 
-    // TODO : 자유 주제로 API를 추가로 구현해보세요.
+    // TODO : 자유 주제로 API를 추가로 구현해보세요. => 관리자가 회원 관리하기 위한 memberId 기준으로 정렬된 list
+    @PostMapping("/members/loginIdAsc")
+    public List<MemberResponse> searchMembersByIdAsc(@RequestParam String memberId) {
+        return memberService.AllMembersWithLoginIdDesc(memberId);
+    }
```

Comment:
코드 잘 작성해주신 것 같습니다.
다만 Post 요청은 보통 데이터를 추가할 때, 또는 변경할 때 사용하므로
GET 요청으로 하는게 조금 더 의미에 와닿을 것 같습니다.

```suggestion
    @GetMapping("/members/loginIdAsc")
    public List<MemberResponse> searchMembersByIdAsc(@RequestParam String memberId) {
        return memberService.AllMembersWithLoginIdDesc(memberId);
    }
```

Reply:
좋은 지적 감사합니다!!